Resources/Prototypes/Entities/Objects/Fun/Instruments/instruments_misc.yml

```diff
@@ -124,4 +124,44 @@
   - type: Clothing
     sprite: Objects/Fun/Instruments/bike_horn.rsi
     slots: [Belt]
-    quickEquip: false
\ No newline at end of file
+    quickEquip: false
+
+- type: entity
+  parent: PhoneInstrument
+  id: BananaPhoneInstrument
+  name: banana phone
+  description: A direct line to the Honkmother. Seems to always go to voicemail.
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/Instruments/otherinstruments.rsi
+    state: banana_phone
+  - type: EmitSoundOnLand
+    sound:
+       path: /Audio/Items/bikehorn.ogg
+  - type: Instrument
```

Comment:
if you are not gonna change the midi sound then just inherit this

Reply:
when you say that are we talking about the type: Instrument program: 124

Reply:
because lines 139-141 are the emit on land

Reply:
> when you say that are we talking about the type: Instrument program: 124

Just realised i can remove this will fix this in the morning

Reply:
i tried inherting this but the tests fail

Reply:
Inheriting is fine, test failure wasnt your fault